cli/dcoscli/package/main.py

```diff
@@ -206,6 +215,197 @@ def _remove_repo(repo_name):
     return 0
 
 
+def _bundle(package_json,
+            output_directory):
+    """ Creates a bundle from a project.
+
+    :param package_json: The package json of the project
+    :type package_json: str
+    :param output_directory: The directory where the bundle will be stored
+    :type output_directory: str
+    :returns: Process status
+    :rtype: int
+    """
+    # get the path of package json
+    cwd = os.getcwd()
+    package_json_path = package_json
+    if not os.path.isabs(package_json_path):
+        package_json_path = os.path.join(cwd, package_json_path)
+
+    package_directory = os.path.dirname(package_json_path)
+
+    # get the path to the output directory
+    if output_directory is None:
+        package_json_dir = os.path.dirname(package_json_path)
+        output_directory = os.path.join(package_json_dir, "target/")
+
+        # create the directory if it does not exist
+        if not (os.path.exists(output_directory)):
+            os.makedirs(output_directory)
```

Comment:
Instead of checking for existence, just use the `exist_ok` parameter:

```python
# ensure the directory exists
os.makedirs(output_directory, exist_ok=True)
```

This avoids a race condition in the current code: if the directory is created by another process after the call to `exists()` returns, then `makedirs()` will raise an exception.